src/hevm/src/EVM/SymExec.hs

```diff
@@ -453,11 +463,11 @@ extractProps = \case
 
 
 -- | Symbolically execute the VM and check all endstates against the postcondition, if available.
-verify :: SolverGroup -> VeriOpts -> VM -> Maybe (Fetch.BlockNumber, Text) -> Maybe Postcondition -> IO (Expr End, [VerifyResult])
-verify solvers opts preState rpcinfo maybepost = do
+verify :: SolverGroup -> VeriOpts -> VM -> Maybe Postcondition -> IO (Expr End, [VerifyResult])
```

Comment:
This is actually cleaner. I like passing it as part of `VeriOpts`. I was always confused what is what